src/hardware/driver/internal_adc.rs

```diff
@@ -1,16 +1,19 @@
-// This is a dummy driver for the Driver analog reads of the output Voltage and Current.
-// Exact Sacales and Pinout will be filled in once we have HW.
-
-use super::Channel;
+use super::{Channel, ChannelVariant};
 use crate::hardware::shared_adc::AdcChannel;
 
 use super::super::hal::{
     gpio::{gpiof::*, Analog},
-    stm32::{ADC1, ADC3},
+    stm32::{ADC1, ADC2, ADC3},
 };
 
 const V_REF: f32 = 2.048; // ADC reference voltage
-const R_SENSE: f32 = 0.1; // Driver output current sense resistor (Will maybe be something else on HW)
+
+// See schematic "diagnostics" sheet. Output voltage sense.
+const V_OFFSET: f32 = 1.; // divider/shifter opamp offset
```

Comment:
Units? For all constants.

Reply:
I have commented the units now. Or did you mean I should be using a unit wrapper for the float value?

Reply:
No. Just spell out the (hopefully SI) units in the comments.

Reply:
Yes, all SI